### PR TITLE
GEOMESA-1314 - Update json4s and scalatra versions

### PR DESCRIPTION
**Pull request by ddseapy**
* Updating json4s 3.2.10 to 3.3.0
* Updating scalatra 2.3.0 to 2.4.0

Signed-off-by: David Seapy <ddseapy@ccri.com>

### CODE REVIEW

File: build/cqs.tsv [hunk 1]
```diff
@@ -148,24 +147,23 @@ org.geotools.xsd:gt-xsd-wfs	14.1	compile
 org.hsqldb:hsqldb	2.3.0	compile
 org.jdom:jdom	1.1.3	compile
 org.joda:joda-convert	1.6	compile
-org.json4s:json4s-ast_2.11	3.2.10	compile
-org.json4s:json4s-core_2.11	3.2.10	compile
-org.json4s:json4s-native_2.11	3.2.10	compile
+org.json4s:json4s-ast_2.11	3.3.0	compile
```
Comment (ddseapy) [inline build/cqs.tsv]: needs cqs for json4s:* 3.3.0 and scalatra:* 2.4.0

**Comment by jnh5y** [comment]
@ddseapy thanks!

@elahrvivaz, are you willing to take a pass at the CQs for this PR?  Also, it'd be good to update the json4s-ext CQ to 2.4.0.

**Comment by jnh5y** [comment]
@elahrvivaz, actually, it might just make sense to enter a json4s 3.3.0 CQ which adds -ext and remove the other CQ.

**Comment by ddseapy** [comment]
I'd hold off on merging this for a bit.  Although I've deployed geomesa-web-data/geomesa-web-install and it appears fine, i'd like to do a bit more testing.  I'm a bit concerned under certain use cases it may hit runtime errors since spark depends on json4s 3.2.10, and we are providing a different version.

**Pull request closed without merging**